backend/handler/BackendHandler.ts

```diff
@@ -227,6 +236,32 @@ export function initWidgetBackends() {
         });
 }
 
+const runScriptInIsolate = async (script: string, query: qs.ParsedQs, headers: IncomingHttpHeaders) => {
+    const isolate = new ivm.Isolate();
+    const context = isolate.createContextSync();
+    const sandbox = context.global;
+
+    // Utility function
+    SandboxService.methodsList.forEach((method: string) => {
+        sandbox.setSync(method, SandboxService[method as keyof typeof SandboxService], { reference: true });
```

Comment:
There's inconsistency in typing. Can we do:
```suggestion
    SandboxService.methodsList.forEach((method: keyof typeof SandboxService) => {
        sandbox.setSync(method, SandboxService[method], { reference: true });
```
?

Reply:
while exposing functions to IVM i need the names as well, hence argument is of type string, for array subsript type string wouldn't work hence need type casting. I tried as you suggested but seems difficult.

Reply:
OK, let's keep it then.